src/routes/v1/checkTxs.ts

```diff
@@ -1,16 +1,18 @@
-import { asArray, asObject, asOptional, asString } from 'cleaners'
+import { asArray, asObject, asOptional, asString, asValue } from 'cleaners'
 import Router from 'express-promise-router'
 
 import { reportsApps, reportsTransactions } from '../../indexApi'
-import { asApps, asDbTx } from '../../types'
+import { asApps, asDbTx, StandardTx } from '../../types'
 
-interface CheckTxsResponse {
+interface CheckTxsResponse extends Partial<StandardTx> {
```

Comment:
The `Partial<StandardTx>` is more ambiguous than:

```ts
interface CheckTxsResponse extends Omit<StandardTx, 'rawTx'>
```

It may not be possible because of a breaking change, but a more straight forward API would look like:

```ts
interface CheckTxsSuccessResponse {
  pluginId: string
  tx: Omit<StandardTx, 'rawTx'>
}

interface CheckTxsFailureResponse {
  pluginId: string
  error: string
}

type CheckTxsResponse = CheckTxsSuccessResponse | CheckTxsFailureResponse
```